satpy/tests/reader_tests/test_hsaf_grib.py

```diff
@@ -0,0 +1,172 @@
+#!/usr/bin/env python
+# -*- coding: utf-8 -*-
+"""Module for testing the satpy.readers.grib module.
```

Comment:
Can you put the license and copyright here ?

Reply:
Done!